docs/src/Guides/03 Creating Commands.md

```diff
@@ -301,8 +301,114 @@ You are in luck. There are currently four different ways to create interactions,
+
+If you do not want admins to be able to overwrite your permissions, or the permissions are not flexible enough for you, you should use [checks][check-this-out].
+
+In this example, we will limit access to the command to members with the `MANAGE_EVENTS` and `MANAGE_THREADS` permissions.
+There are two ways to define permissions.
+
+=== ":one: Decorators"
+    ```py
+    @slash_command(name="my_command")
+    @slash_default_member_permission(Permissions.MANAGE_EVENTS)
+    @slash_default_member_permission(Permissions.MANAGE_THREADS)
+    async def my_command_function(ctx: InteractionContext):
+        ...
+    ```
+
+=== ":two: Function Definition"
+    ```py
+    @slash_command(
+        name="my_command",
+        default_member_permissions=Permissions.MANAGE_EVENTS | Permissions.MANAGE_THREADS,
+    )
+    async def my_command_function(ctx: InteractionContext):
+        ...
+    ```
+
+Multiple permissions are defined with the bitwise OR operator `|`.
+
+### Blocking Commands in DMs
+
+You can also block commands in DMs. To do that, just set `dm_permission` to false.
+
+```py
+@slash_command(
+    name="my_guild_only_command",
+    dm_permission=False,
+)
+async def my_command_function(ctx: InteractionContext):
+    ...
+```
+
+### Context Menus
+
+Both default permissions and DM blocking can be used the same way for context menus, since they are normal slash commands under the hood.
+
+### Check This Out
+
+Checks allow you to define who can use your commands however you want.
+
+There are a few pre-made checks for you to use, and you can simply create your own custom checks.
+
+=== "Build-In Check"
+    Check that the author is the owner of the bot:  
+
+    ```py
+    @is_owner()
+    @slash_command(name="my_command")
+    async def my_command_function(ctx: InteractionContext):
+        ...
+    ```
+
+=== "Custom Check"
+    Check that the author's name starts with `a`:  
+
+    ```py
+    async def my_check(ctx: Context):
+        return ctx.author.name.startswith("a")
+
+    @check(check=my_check)
+    @slash_command(name="my_command")
+    async def my_command_function(ctx: InteractionContext):
+        ...
+    ```
+
+=== "Reusing Checks"
+    You can reuse checks in extensions by adding them to the extension check list
+
+    ```py
+    class MyExtension(Scale):
+        def __init__(self, bot) -> None:
+            super().__init__(bot)
+            self.add_scale_check(is_owner())
+
+    @slash_command(name="my_command")
+    async def my_command_function(ctx: InteractionContext):
+        ...
+
+    @slash_command(name="my_command2")
+    async def my_command_function2(ctx: InteractionContext):
+        ...
+
+    def setup(bot) -> None:
+        MyExtension(bot)
+    ```
+
+    The check will be checked for every command in the extension.
+
+This example only allows the bot owner to execute a command.
+```py
+
+
+```
```

Comment:
I think you forgot to put in the actual example 😅

Reply:
Woops